README.md

```diff
@@ -1 +1,147 @@
-# hello-world-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
```

Comment:
- [ ] [Optional] Kindly remove this section as it is not needed in your readme file. You already have the table of content below.